ryannhg/elm-in-the-spring
# Allow us to link directly to speakers' dialogs

We want to be able to post tweets with links directly to the info about a speaker's talk -- this PR does that!

It'd be nice to have a URL like `/speaker/NAME`, but that doesn't work (at least in local dev) because of how we're serving the app. Instead, we use a query parameter.

**Test steps**

Try it out!

http://localhost:5000/?speaker=BrianHicks
http://localhost:5000/?speaker=KatieHughes
http://localhost:5000/?speaker=RichardFeldman

Note that the query parameter remains even if you close the dialog; IMO that's okay, but we can fix that if desired.
